org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/src/main/java/org/eclipse/che/jdt/ls/extension/core/internal/JavaModelUtil.java

```diff
@@ -59,6 +53,19 @@ public static IJavaProject getJavaProject(String resourceUri) {
     return JavaCore.create(project);
   }
 
+  /**
+   * Returns all user created java project which exist in current workspace. This method excludes
+   * default {@code jdt.ls-java-project} project.
+   *
+   * @return all user created java projects in current workspace
+   */
+  public static List<IJavaProject> getWorkspaceJavaProjects() {
+    return Arrays.stream(ResourcesPlugin.getWorkspace().getRoot().getProjects())
+        .filter(project -> !project.getName().equals("jdt.ls-java-project"))
```

Comment:
As for me would be better if `jdt.ls-java-project`  would be declared as constant. And for equals operation use a constant firstly than an object with which to compare to avoid NPE.